src/types/index.ts

```diff
@@ -1,14 +1,24 @@
+import { Problem } from "../problem";
+
 export type ProblemInterface = {
-    http?: httpObject,
-    type: string,
-    title: string,          // Title should be description of http status, if type is not present.
-    detail?: string,
-    instance?: string,      // Details to reproduce the error.
-    stack?: string;
-    [key: string]: any,      // Custom Field of Problem
-}
-
-export type httpObject= {
-    status: number,          // Status Code
-    [key: string]: any,
+  type: string;
+  title: string; // Title should be description of http status, if type is not present.
+  http?: httpObject;
+  detail?: string;
+  instance?: string; // Details to reproduce the error.
```

Comment:
That description should probably points to the `detail` field or am I wrong? 🤔

Reply:
Let's keep `detail` and remove `title`. @magicmatatjahu